### PR TITLE
doc: 3.23.0 Changelog

### PR DESCRIPTION
**Pull request by chenshuai2144**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

## 3.23.0

`2019-09-02`

- 🔥 Pageheader has been redesigned to accommodate more situations. [#18128](https://github.com/ant-design/ant-design/pull/18128)
- 🌟 Card support `tabBarExtraContent` prop. [#18433](https://github.com/ant-design/ant-design/pull/18433) [@lengthmin](https://github.com/lengthmin)
- 🌟 Anchor.Link add add new property `target`. [#18335](https://github.com/ant-design/ant-design/pull/18335) [@DiamondYuan](https://github.com/DiamondYuan)
- 🌟 Breadcrumb support children use with `React.Fragment`. [#18340](https://github.com/ant-design/ant-design/pull/18340) [@long-zhuge](https://github.com/long-zhuge)
- 🌟 Card.Grid add new prop `hoverable` for support disable hover state. [#18457](https://github.com/ant-design/ant-design/pull/18457) [@MrHeer](https://github.com/MrHeer)
- 🇦🇲 Added locales for the Armenian language. [#18586](https://github.com/ant-design/ant-design/pull/18586) [@ashmna](https://github.com/ashmna)
- Less variables
  - 💄 Add `@modal-footer-border-color-split`. [#18522](https://github.com/ant-design/ant-design/pull/18522)
  - 💄 Add `@input-number-handler-bg` `@input-number-handler-border-color`. [#18533](https://github.com/ant-design/ant-design/pull/18533)
  - 💄 Add `@card-background` `@card-skeleton-bg`. [#18531](https://github.com/ant-design/ant-design/pull/18531)
  - 💄 Add `@typography-title-font-weight`. [#18456](https://github.com/ant-design/ant-design/pull/18456) [@MrHeer](https://github.com/MrHeer)
- 🛎 Add warning when use `value` of Switch, Checkbox, Upload. [#18497](https://github.com/ant-design/ant-design/pull/18497)
- 🐞 Fix circle Button not circle in some situation. [#18516](https://github.com/ant-design/ant-design/pull/18516)
- 🐞 Fix icon position in Input. [#18521](https://github.com/ant-design/ant-design/pull/18521) [@xrkffgg](https://github.com/xrkffgg)
- 🐞 Fix RangePicker selected date style. [#18559](https://github.com/ant-design/ant-design/pull/18559)
- Descriptions
  - 🐞 Fix Descriptions last Item has wrong calculated width issue. [#18568](https://github.com/ant-design/ant-design/pull/18568)
  - 🐞 Description.Item will reuse key in render if user provided. [#18578](https://github.com/ant-design/ant-design/pull/18578)
- 🐞 Fix Tab content width not correct in Safari. [#18574](https://github.com/ant-design/ant-design/pull/18574)
- 🐞 Fix Mentions popup position not correct when `prefix` is empty string. [#18576](https://github.com/ant-design/ant-design/pull/18576)
- 🐞 Fix Upload.Dragger can upload multiple files when multiple is false. [#18580](https://github.com/ant-design/ant-design/pull/18580)
- 🐞 Fix Card `actions` style when contains link Button. [#18588](https://github.com/ant-design/ant-design/pull/18588)
- 🐞 Fix Input not vertical align with Button in Chrome. [#18603](https://github.com/ant-design/ant-design/pull/18603)
- 🐞 Fix nested List style in grid layout. [#18589](https://github.com/ant-design/ant-design/pull/18589)
- TypeScript
  - 🐞 Fix Steps.Step component's `subTitle` prop types. [#18525](https://github.com/ant-design/ant-design/pull/18525) [@wtzeng1](https://github.com/wtzeng1)
  - 🐞 Ensure title or overlay is required in Tooltip props definition. [#18515](https://github.com/ant-design/ant-design/pull/18515) [@laysent](https://github.com/laysent)

## 3.23.0

`2019-09-02`

- 🔥 PageHeader 样式重新设计以适应更多的情况。[#18128](https://github.com/ant-design/ant-design/pull/18128)
- 🌟 Card 组件新增了 `tabBarExtraContent` 属性。[#18433](https://github.com/ant-design/ant-design/pull/18433) [@lengthmin](https://github.com/lengthmin)
- 🌟 Anchor.Link 增加 `target` 属性。[#18335](https://github.com/ant-design/ant-design/pull/18335) [@DiamondYuan](https://github.com/DiamondYuan)
- 🌟 Breadcrumb 支持子组件使用 `React.Fragment` 的场景。[#18340](https://github.com/ant-design/ant-design/pull/18340) [@long-zhuge](https://github.com/long-zhuge)
- 🌟 Card.Grid 新增 `hoverable` 属性允许禁用浮动效果。[#18457](https://github.com/ant-design/ant-design/pull/18457) [@MrHeer](https://github.com/MrHeer)
- 🇦🇲 新增亚美尼亚语。[#18586](https://github.com/ant-design/ant-design/pull/18586) [@ashmna](https://github.com/ashmna)
- Less 变量
  - 💄 增加 `@modal-footer-border-color-split`。[#18522](https://github.com/ant-design/ant-design/pull/18522)
  - 💄 增加 `@input-number-handler-bg` `@input-number-handler-border-color`。[#18533](https://github.com/ant-design/ant-design/pull/18533)
  - 💄 增加 `@card-background` `@card-skeleton-bg`。[#18531](https://github.com/ant-design/ant-design/pull/18531)
  - 💄 增加 `@typography-title-font-weight`。[#18456](https://github.com/ant-design/ant-design/pull/18456) [@MrHeer](https://github.com/MrHeer)
- 🛎 Switch、Checkbox 和 Upload 使用 `value` 时会提示正确的属性名。[#18497](https://github.com/ant-design/ant-design/pull/18497)
- 🐞 修复圆形 Button 有时候不是圆形的问题。[#18516](https://github.com/ant-design/ant-design/pull/18516)
- 🐞 修复 Input 中图标位置不居中情况。[#18521](https://github.com/ant-design/ant-design/pull/18521) [@xrkffgg](https://github.com/xrkffgg)
- 🐞 修复 RangePicker 多余的选中日期样式。[#18559](https://github.com/ant-design/ant-design/pull/18559)
- Descriptions
  - 🐞 修复 Descriptions.Item 最后个宽度计算不正确的问题。[#18568](https://github.com/ant-design/ant-design/pull/18568)
  - 🐞 Description.Item 在渲染时会复用用户提供的 key。[#18578](https://github.com/ant-design/ant-design/pull/18578)
- 🐞 修复 Tab 内容宽度在 Safari 下不正确的问题。[#18574](https://github.com/ant-design/ant-design/pull/18574)
- 🐞 修复 Mentions 的 prefix 为空字符串时，弹窗位置不正确的问题。[#18576](https://github.com/ant-design/ant-design/pull/18576)
- 🐞 修复 Upload.Dragger 在 multiple 为 false 时，仍然可以上传多份文件的问题。[#18580](https://github.com/ant-design/ant-design/pull/18580)
- 🐞 修复 `Button[href]` 在 Card `actions` 中样式变形的问题。[#18588](https://github.com/ant-design/ant-design/pull/18588)
- 🐞 修复 Chrome 中 Input 与 Button 不垂直对齐的问题。[#18603](https://github.com/ant-design/ant-design/pull/18603)
- 🐞 修复网格 List 中内嵌 List 的边距问题。[#18589](https://github.com/ant-design/ant-design/pull/18589)
- TypeScript
  - 🐞 修复 Steps.Step 组件 `subTitle` 属性类型。[#18525](https://github.com/ant-design/ant-design/pull/18525) [@wtzeng1](https://github.com/wtzeng1)
  - 🐞 确保 Tooltip 属性定义中，`title` 或 `overlay` 至少有一个是必填的。[#18515](https://github.com/ant-design/ant-design/pull/18515) [@laysent](https://github.com/laysent)

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/changelog/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/changelog/CHANGELOG.zh-CN.md)

### CODE REVIEW

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -15,6 +15,16 @@ timeline: true
 
 ---
 
+## 3.23.0
+
+- 🔥 PageHeader 样式重写以适应更多的情况。 [#18128](https://github.com/ant-design/ant-design/pull/18128)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 不是应该发 3.22.1 么

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,16 @@ timeline: true
...
+- 💄 调整 Table `expand icon` 的样式。 [c5344bd](https://github.com/ant-design/ant-design/commit/c5344bde529a2f2ec814f46e7ec5d249eac8d608)
+- 🌟 Card 组件新增了 `tabBarExtraContent` props。 [#18433](https://github.com/ant-design/ant-design/pull/18433) [@lengthmin](https://github.com/lengthmin)
+- 🌟 Anchor.Link 增加 `target` 属性。 [#18335](https://github.com/ant-design/ant-design/pull/18335) [@DiamondYuan](https://github.com/DiamondYuan)
+- 🐞 修复 Table 的 `style` 属性被应用了两次的问题. [#18330](https://github.com/ant-design/ant-design/pull/18330) [@MrHeer](https://github.com/MrHeer)
+- 🐞 修复 Input 在 IE 下错位的问题。 [#17759](https://github.com/ant-design/ant-design/pull/17759)
+- 🐞 修复 Steps `progressDot` 样式错位问题。 [#18356](https://github.com/ant-design/ant-design/pull/18356)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 句号后面多了空格。

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,16 @@ timeline: true
...
 ---
 
+## 3.23.0
+
+- 🔥 PageHeader 样式重写以适应更多的情况。 [#18128](https://github.com/ant-design/ant-design/pull/18128)
+- 💄 调整 Table `expand icon` 的样式。 [c5344bd](https://github.com/ant-design/ant-design/commit/c5344bde529a2f2ec814f46e7ec5d249eac8d608)
+- 🌟 Card 组件新增了 `tabBarExtraContent` props。 [#18433](https://github.com/ant-design/ant-design/pull/18433) [@lengthmin](https://github.com/lengthmin)
+- 🌟 Anchor.Link 增加 `target` 属性。 [#18335](https://github.com/ant-design/ant-design/pull/18335) [@DiamondYuan](https://github.com/DiamondYuan)
+- 🐞 修复 Table 的 `style` 属性被应用了两次的问题. [#18330](https://github.com/ant-design/ant-design/pull/18330) [@MrHeer](https://github.com/MrHeer)
+- 🐞 修复 Input 在 IE 下错位的问题。 [#17759](https://github.com/ant-design/ant-design/pull/17759)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: IE11

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,16 @@ timeline: true
 
 ---
 
+## 3.23.0
+
+- 🔥 PageHeader 样式重写以适应更多的情况。 [#18128](https://github.com/ant-design/ant-design/pull/18128)
+- 💄 调整 Table `expand icon` 的样式。 [c5344bd](https://github.com/ant-design/ant-design/commit/c5344bde529a2f2ec814f46e7ec5d249eac8d608)
+- 🌟 Card 组件新增了 `tabBarExtraContent` props。 [#18433](https://github.com/ant-design/ant-design/pull/18433) [@lengthmin](https://github.com/lengthmin)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 不要中英文混用。

File: CHANGELOG.zh-CN.md [hunk 5]
```diff
@@ -15,6 +15,16 @@ timeline: true
 
 ---
 
+## 3.23.0
+
+- 🔥 PageHeader 样式重写以适应更多的情况。 [#18128](https://github.com/ant-design/ant-design/pull/18128)
+- 💄 调整 Table `expand icon` 的样式。 [c5344bd](https://github.com/ant-design/ant-design/commit/c5344bde529a2f2ec814f46e7ec5d249eac8d608)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: expand icon 

不要中英文混用。

File: CHANGELOG.en-US.md [hunk 6]
```diff
@@ -15,6 +15,16 @@ timeline: true
 
 ---
 
+## 3.23.0
+
+- 🔥 Pageheader has been rewritten to accommodate more situations. [#18128](https://github.com/ant-design/ant-design/pull/18128)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: redesigned

**Comment by afc163** [comment]
- [ ] https://github.com/ant-design/ant-design/pull/18425

File: CHANGELOG.zh-CN.md (hunk 1 above)
Comment (chenshuai2144) [reply CHANGELOG.zh-CN.md]: PageHeader 的样式重写有一个 breakchange ，背景颜色被取消了

Comment (afc163) [reply CHANGELOG.zh-CN.md]: PageHeader 为啥要发，月底统一不行么。

**Comment by afc163** [comment]
建议月底发 3.23.0，本周发 3.22.1 就好。

**Comment by afc163** [comment]
- [ ] https://github.com/ant-design/ant-design/pull/18458

File: CHANGELOG.zh-CN.md (hunk 1 above)
Comment (chenshuai2144) [reply CHANGELOG.zh-CN.md]: techui 有点着急要，有几个模板依赖他。

File: CHANGELOG.en-US.md [hunk 7]
```diff
@@ -15,6 +15,35 @@ timeline: true
 
 ---
 
+## 3.23.0
+
+`2019-09-02`
+
+- 🔥 Pageheader has been redesigned to accommodate more situations. [#18128](https://github.com/ant-design/ant-design/pull/18128)
+- 🌟 Add `tabBarExtraContent` property to Card component. [#18433](https://github.com/ant-design/ant-design/pull/18433) [@lengthmin](https://github.com/lengthmin)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: Card support `tabBarExtraContent` prop.

**Review by zombieJ — APPROVED** [review]

**Review by dengfuping — APPROVED** [review]

File: CHANGELOG.en-US.md [hunk 8]
```diff
@@ -15,6 +15,37 @@ timeline: true
...
+- 🌟 Card support `tabBarExtraContent` prop. [#18433](https://github.com/ant-design/ant-design/pull/18433) [@lengthmin](https://github.com/lengthmin)
+- 🌟 Anchor.Link add add new property `target`. [#18335](https://github.com/ant-design/ant-design/pull/18335) [@DiamondYuan](https://github.com/DiamondYuan)
+- 🌟 Breadcrumb support children use with `React.Fragment`. [#18340](https://github.com/ant-design/ant-design/pull/18340) [@long-zhuge](https://github.com/long-zhuge)
+- 🌟 Card.Grid add new prop `hoverable` for support disable hover state. [#18457](https://github.com/ant-design/ant-design/pull/18457) [@MrHeer](https://github.com/MrHeer)
+- 🇦🇲 Added locales for the Armenian language. [#18586](https://github.com/ant-design/ant-design/pull/18586) [@ashmna](https://github.com/ashmna)
+- 💄 Add less variables `@modal-footer-border-color-split`. [#18522](https://github.com/ant-design/ant-design/pull/18522)
+- 💄 Add less variables `@input-number-handler-bg` `@input-number-handler-border-color`. [#18533](https://github.com/ant-design/ant-design/pull/18533)
+- 💄 Add less variables `@card-backgroun` `@card-skeleton-bg`. [#18531](https://github.com/ant-design/ant-design/pull/18531)
```
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: `@card-backgroun` -> `@card-background`

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/18589
https://github.com/ant-design/ant-design/pull/18603

**Comment by yutingzhao1991** [comment]
@afc163 这两个都有了的

File: CHANGELOG.zh-CN.md [hunk 9]
```diff
@@ -15,6 +15,37 @@ timeline: true
...
+- 🌟 Anchor.Link 增加 `target` 属性。[#18335](https://github.com/ant-design/ant-design/pull/18335) [@DiamondYuan](https://github.com/DiamondYuan)
+- 🌟 Breadcrumb 支持子组件使用 `React.Fragment` 的场景。[#18340](https://github.com/ant-design/ant-design/pull/18340) [@long-zhuge](https://github.com/long-zhuge)
+- 🌟 Card.Grid 新增 `hoverable` 属性允许禁用浮动效果。[#18457](https://github.com/ant-design/ant-design/pull/18457) [@MrHeer](https://github.com/MrHeer)
+- 🇦🇲 新增亚美尼亚语。[#18586](https://github.com/ant-design/ant-design/pull/18586) [@ashmna](https://github.com/ashmna)
+- 💄 增加 less 变量 `@modal-footer-border-color-split`。[#18522](https://github.com/ant-design/ant-design/pull/18522)
+- 💄 增加 less 变量 `@input-number-handler-bg` `@input-number-handler-border-color`。[#18533](https://github.com/ant-design/ant-design/pull/18533)
+- 💄 增加 less 变量 `@card-background` `@card-skeleton-bg`。[#18531](https://github.com/ant-design/ant-design/pull/18531)
+- 💄 增加 less 变量 `@typography-title-font-weight`。[#18456](https://github.com/ant-design/ant-design/pull/18456) [@MrHeer](https://github.com/MrHeer)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 应该合并写，类似 TypeScript

File: CHANGELOG.zh-CN.md [hunk 10]
```diff
@@ -15,6 +15,37 @@ timeline: true
...
+  - 🐞 Description.Item 在渲染时会复用用户提供的 key。[#18578](https://github.com/ant-design/ant-design/pull/18578)
+- 🐞 修复 Tab 内容宽度在 Safari 下不正确的问题。[#18574](https://github.com/ant-design/ant-design/pull/18574)
+- 🐞 修复 Mentions 的 prefix 为空字符串时，弹窗位置不正确的问题。[#18576](https://github.com/ant-design/ant-design/pull/18576)
+- 🐞 修复 Upload.Dragger 在 multiple 为 false 时，仍然可以上传多份文件的问题。[#18580](https://github.com/ant-design/ant-design/pull/18580)
+- 🐞 修复 `Button[href]` 在 Card `actions` 中样式变形的问题。[#18588](https://github.com/ant-design/ant-design/pull/18588)
+- 🐞 修复 Chrome 中 Input 与 Button 不垂直对齐的问题。[#18603](https://github.com/ant-design/ant-design/pull/18603)
+- 🐞 修复网格 List 中内嵌 List 的边距问题。[#18589](https://github.com/ant-design/ant-design/pull/18589)
+- TypeScript
+  - 🐞 修复 Steps.Step 组件 `subTitle` 属性类型。[#18525](https://github.com/ant-design/ant-design/pull/18525) [@wtzeng1](https://github.com/wtzeng1)
+  - 🐞 确保 Tooltip props 定义中，`title` 或 `overlay` 至少有一个是必填的。[#18515](https://github.com/ant-design/ant-design/pull/18515) [@laysent](https://github.com/laysent)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: props => 属性

File: CHANGELOG.zh-CN.md [hunk 11]
```diff
@@ -15,6 +15,37 @@ timeline: true
...
+- 🌟 Card.Grid 新增 `hoverable` 属性允许禁用浮动效果。[#18457](https://github.com/ant-design/ant-design/pull/18457) [@MrHeer](https://github.com/MrHeer)
+- 🇦🇲 新增亚美尼亚语。[#18586](https://github.com/ant-design/ant-design/pull/18586) [@ashmna](https://github.com/ashmna)
+- 💄 增加 less 变量 `@modal-footer-border-color-split`。[#18522](https://github.com/ant-design/ant-design/pull/18522)
+- 💄 增加 less 变量 `@input-number-handler-bg` `@input-number-handler-border-color`。[#18533](https://github.com/ant-design/ant-design/pull/18533)
+- 💄 增加 less 变量 `@card-background` `@card-skeleton-bg`。[#18531](https://github.com/ant-design/ant-design/pull/18531)
+- 💄 增加 less 变量 `@typography-title-font-weight`。[#18456](https://github.com/ant-design/ant-design/pull/18456) [@MrHeer](https://github.com/MrHeer)
+- 🛎 Switch、Checkbox 和 Upload 使用 value 时会提示正确的属性名。[#18497](https://github.com/ant-design/ant-design/pull/18497)
+- 🐞 修复圆形 Button 有时候不是圆形的问题。[#18516](https://github.com/ant-design/ant-design/pull/18516)
+- 🐞 修复 Input 中 icon 位置不居中情况。[#18521](https://github.com/ant-design/ant-design/pull/18521) [@xrkffgg](https://github.com/xrkffgg)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: icon => 图标

File: CHANGELOG.zh-CN.md [hunk 12]
```diff
@@ -15,6 +15,37 @@ timeline: true
...
+- 🌟 Anchor.Link 增加 `target` 属性。[#18335](https://github.com/ant-design/ant-design/pull/18335) [@DiamondYuan](https://github.com/DiamondYuan)
+- 🌟 Breadcrumb 支持子组件使用 `React.Fragment` 的场景。[#18340](https://github.com/ant-design/ant-design/pull/18340) [@long-zhuge](https://github.com/long-zhuge)
+- 🌟 Card.Grid 新增 `hoverable` 属性允许禁用浮动效果。[#18457](https://github.com/ant-design/ant-design/pull/18457) [@MrHeer](https://github.com/MrHeer)
+- 🇦🇲 新增亚美尼亚语。[#18586](https://github.com/ant-design/ant-design/pull/18586) [@ashmna](https://github.com/ashmna)
+- 💄 增加 less 变量 `@modal-footer-border-color-split`。[#18522](https://github.com/ant-design/ant-design/pull/18522)
+- 💄 增加 less 变量 `@input-number-handler-bg` `@input-number-handler-border-color`。[#18533](https://github.com/ant-design/ant-design/pull/18533)
+- 💄 增加 less 变量 `@card-background` `@card-skeleton-bg`。[#18531](https://github.com/ant-design/ant-design/pull/18531)
+- 💄 增加 less 变量 `@typography-title-font-weight`。[#18456](https://github.com/ant-design/ant-design/pull/18456) [@MrHeer](https://github.com/MrHeer)
+- 🛎 Switch、Checkbox 和 Upload 使用 value 时会提示正确的属性名。[#18497](https://github.com/ant-design/ant-design/pull/18497)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: value 加 `

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/18346

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/18541

File: CHANGELOG.en-US.md [hunk 13]
```diff
@@ -15,6 +15,38 @@ timeline: true
...
+- 🛎 Add warning when use `value` of Switch, Checkbox, Upload. [#18497](https://github.com/ant-design/ant-design/pull/18497)
+- 🐞 Fix circle Button not circle in some situation. [#18516](https://github.com/ant-design/ant-design/pull/18516)
+- 🐞 Fix icon position in Input. [#18521](https://github.com/ant-design/ant-design/pull/18521) [@xrkffgg](https://github.com/xrkffgg)
+- 🐞 Fix RangePicker selected date style. [#18559](https://github.com/ant-design/ant-design/pull/18559)
+- Descriptions
+  - 🐞 Fix Descriptions last Item has wrong calculated width issue. [#18568](https://github.com/ant-design/ant-design/pull/18568)
+  - 🐞 Description.Item will reuse key in render if user provided. [#18578](https://github.com/ant-design/ant-design/pull/18578)
+- 🐞 Fix Tab content width not correct in Safari. [#18574](https://github.com/ant-design/ant-design/pull/18574)
+- 🐞 Fix Mentions popup position not correct when `prefix` is empty string. [#18576](https://github.com/ant-design/ant-design/pull/18576)
+- 🐞 Fix Upload.Dragger can upload multiple files when multiple is false. [#18580](https://github.com/ant-design/ant-design/pull/18580)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: 后一个 multiple 加 `

属性名都要加 `

File: CHANGELOG.zh-CN.md [hunk 14]
```diff
@@ -39,7 +43,7 @@ timeline: true
   - 🐞 Description.Item 在渲染时会复用用户提供的 key。[#18578](https://github.com/ant-design/ant-design/pull/18578)
 - 🐞 修复 Tab 内容宽度在 Safari 下不正确的问题。[#18574](https://github.com/ant-design/ant-design/pull/18574)
 - 🐞 修复 Mentions 的 prefix 为空字符串时，弹窗位置不正确的问题。[#18576](https://github.com/ant-design/ant-design/pull/18576)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 属性名都要加 `

File: CHANGELOG.zh-CN.md [hunk 15]
```diff
@@ -42,7 +42,7 @@ timeline: true
   - 🐞 修复 Descriptions.Item 最后个宽度计算不正确的问题。[#18568](https://github.com/ant-design/ant-design/pull/18568)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 最后一个

File: CHANGELOG.zh-CN.md [hunk 16]
```diff
@@ -42,7 +42,7 @@ timeline: true
...
   - 🐞 Description.Item 在渲染时会复用用户提供的 key。[#18578](https://github.com/ant-design/ant-design/pull/18578)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: `

**Pull request merged**